.expeditor/announce-acceptance.sh

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
 
-current=$(curl https://packages.chef.io/manifests/current/automate/latest.json 2>/dev/null | jq -r '.git_sha');
-acceptance=$(curl https://packages.chef.io/manifests/acceptance/automate/latest.json 2>/dev/null | jq -r '.git_sha');
+# Get current sha, then find the one right after that via the ancestry-path; that is where we start.
+current_sha=$(curl https://packages.chef.io/manifests/current/automate/latest.json 2>/dev/null | jq -r '.git_sha');
+current_date=$(git log -1 --pretty=%cI "$(git rev-list --ancestry-path "$current_sha"..HEAD | tail -1)")
+
+# Get the date of the latest acceptance sha and add one second to it, otherwise it does not appear in the list!
+acceptance_sha=$(curl https://packages.chef.io/manifests/acceptance/automate/latest.json 2>/dev/null | jq -r '.git_sha');
+acceptance_date_tmp=$(git log -1 --pretty=%cI "$acceptance_sha")
+acceptance_date=$(date -d "$acceptance_date_tmp + 1 second" '+%Y-%m-%dT%H:%M:%S%z')
```

Comment:
This command didn't work in macOS in general since `date` in macOS is different from `date` in Unix bash version/
This works in mac if we `gdate` instead of `date`, assuming that gnu core utils is installed.

Reply:
Please see my comments in the introduction--I already addressed the mac/linux issue. This is supposed to run on Linux, but I showed how one could test it on mac as well.

Reply:
Yes. I tested by the way you mentioned for mac and it worked fine.
But the final changes won't work perfectly on Mac because we are using `date`.

Although having said that, I believe this is meant to run in Buildkite agents which are linux machines.
Am I correct?